pelicanconf.py

```diff
@@ -1,3 +1,27 @@
 # -*- coding: utf-8 -*-
 
+AUTHOR = 'Komitywa'
+SITENAME = 'Komitywa.org'
+SITEURL = ''
+
+THEME = 'themes/notmyidea'
+
+PATH = 'content'
+
 TIMEZONE = 'Europe/Warsaw'
+
+DEFAULT_LANG = 'pl'
+
+# Feed generation off, since no articles right now
+FEED_ALL_ATOM = None
+CATEGORY_FEED_ATOM = None
+TRANSLATION_FEED_ATOM = None
+AUTHOR_FEED_ATOM = None
+AUTHOR_FEED_RSS = None
+
+
+SOCIAL = (('Facebook', 'https://www.facebook.com/KOMITYWA.org'),)
```

Comment:
I don't know if we are using this settings somewhere, but if we want to define it here, that will be great to add here github also (see: http://docs.getpelican.com/en/3.6.3/settings.html#example-settings).